frontend/src/components/AdmissionCountDown/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import moment from "moment-timezone";
+
+class AdmissionCountDown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      timeDiff: this.calcTimeDiff()
+    };
+  }
+
+  counterInterval;
+  endTime = moment(this.props.endTime);
+
+  componentDidMount() {
+    this.counterInterval = setInterval(
+      () =>
+        this.setState({
+          timeDiff: this.calcTimeDiff()
+        }),
+      1000
+    );
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.counterInterval);
+  }
+
+  calcTimeDiff() {
+    const now = moment();
```

Comment:
Move this constant outside the function so you don't have to declare another similar constant later :)